src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,5 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user.name}
+  </a>
```

Comment:
If there is no user, the link to the email will not work for you, you need to add checks, you can also use destructuring for the user